.github/workflows/e2e-dev.yml

```diff
@@ -22,7 +22,6 @@ defaults:
 
 jobs:
     compute-dev-version:
-        if: "!github.event.push.repository.fork && (github.event.pull_request.draft == false || github.action == 'workflow_dispatch')"
```

Comment:
Why have these conditions been removed?

Reply:
I [removed this temporary](https://github.com/dhis2/data-visualizer-app/pull/2744/commits/9b2fcc88b003a5eb49b01e7443352fe42c481036) …I was not sure why the e2e test was skipped .. so with this change I was able to see the e2e tests running... I already asked the analytics team on slack about it